code/iaas/logic/src/main/java/io/cattle/platform/process/containerevent/ContainerEventCreate.java

```diff
@@ -0,0 +1,197 @@
+package io.cattle.platform.process.containerevent;
+
+import static io.cattle.platform.core.model.tables.InstanceTable.*;
+import io.cattle.platform.archaius.util.ArchaiusUtil;
+import io.cattle.platform.core.constants.InstanceConstants;
+import io.cattle.platform.core.constants.NetworkConstants;
+import io.cattle.platform.core.dao.AccountDao;
+import io.cattle.platform.core.dao.NetworkDao;
+import io.cattle.platform.core.model.ContainerEvent;
+import io.cattle.platform.core.model.Image;
+import io.cattle.platform.core.model.Instance;
+import io.cattle.platform.core.model.Network;
+import io.cattle.platform.engine.handler.HandlerResult;
+import io.cattle.platform.engine.process.ProcessInstance;
+import io.cattle.platform.engine.process.ProcessState;
+import io.cattle.platform.engine.process.impl.ProcessCancelException;
+import io.cattle.platform.lock.LockCallbackNoReturn;
+import io.cattle.platform.lock.LockManager;
+import io.cattle.platform.object.util.DataAccessor;
+import io.cattle.platform.object.util.DataUtils;
+import io.cattle.platform.object.util.ObjectUtils;
+import io.cattle.platform.process.base.AbstractDefaultProcessHandler;
+import io.cattle.platform.storage.service.StorageService;
+import io.cattle.platform.util.exception.ExecutionException;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import org.apache.commons.lang3.BooleanUtils;
+
+import com.netflix.config.DynamicBooleanProperty;
+
+@Named
+public class ContainerEventCreate extends AbstractDefaultProcessHandler {
+
+    private static final DynamicBooleanProperty MANAGE_NONRANCHER_CONTAINERS = ArchaiusUtil
```

Comment:
@ibuildthecloud I currently have the feature turned off by default and you would have to change a setting to turn it on. Would you rather it be on by default?